aion/web/elm/src/Urls.elm

```diff
@@ -25,7 +25,7 @@ questionsUrl location =
 
 roomsUrl : Location -> String
 roomsUrl location =
-    (host location) ++ "api/rooms"
+    (host location) ++ "api/counts"
```

Comment:
Update url to match new one?

Reply:
Yeah, forgot to do that, no tests on frontend make it easy to go through CI check with code that doesn't work at all :joy: